internal/engine/interpreter/interpreter.go

```diff
@@ -166,6 +169,8 @@ type callFrame struct {
 	pc uint64
 	// f is the compiled function used in this function frame.
 	f *function
+	// base index in the frame of this function.
+	base int
```

Comment:
Couldn't figure out a way to detect how many values may be on the stack at the time of a call, so I added a sort of base pointer for this, but maybe I missed an invariant of correct WASM code that makes it unnecessary?

Reply:
I guess this is the only way to get base so I think it's ok to define `base int` here 👍